tests/console/postgresql_server.pm

```diff
@@ -14,13 +14,13 @@ use base "consoletest";
 use strict;
 use testapi;
 use utils;
-use version_utils 'sle_version_at_least';
+use version_utils 'is_sle';
 use apachetest;
 
 sub run {
     select_console 'root-console';
 
-    my $pgsql_server = sle_version_at_least('15') ? 'postgresql10-server' : 'postgresql96-server';
+    my $pgsql_server = (is_sle('15+') or is_sle('>12-sp3')) ? 'postgresql10-server' : 'postgresql96-server';
```

Comment:
will break Leap maintenance tests: https://openqa.opensuse.org/tests/781638#step/postgresql_server/2

Better phrase it the other way around: `(is_sle('<15') or is_leap('<15.0')) ? 'postgresql96-server' : 'postgresql10-server';`

Reply:
`(is_sle('<15') or is_leap('<15.0')) ? 'postgresql96-server' : 'postgresql10-server'; ` is also wrong 

situation is more complicated with SLE12SP4 on way :D

Reply:
And we can't simply zypper in postgresql-server ?

Reply:
@coolo no, this isnt aviable on SLE12, on other hand postgresql10 is was released for all SLE12 servicepacks ..